WEB_SERVICE/WEB/_HIGH/GRAPHQL_DIRECTIVE_OVERLOADING/recommendation.md

```diff
@@ -0,0 +1,58 @@
+To mitigate the risk of Directive Overloading attacks, you can take the following steps:
+
+- **Rate Limiting**: Apply rate-limiting to prevent a large volume of requests from overwhelming your server with excessive directive usage.
+
+- **Implement timouts**: Implementing timeouts to `kill` queries that take too long to resolve can be highly effective. By setting a maximum execution time, you can automatically terminate queries that exceed this limit, preventing them from consuming excessive server resources. 
+
+- **Limit Directives**: Implement server-side limits on the number of directives allowed in a single GraphQL query. You can configure tools like `GraphQL Armor` to limit directives and prevent overloading.
+
+=== "JavaScript"
+  ```javascript
+  // Configuring for GraphQL Armor
+  GraphQLArmorConfig({
+    maxDirectives: {
+      // Toogle the plugin | default: true
+      enabled?: boolean,
+    
+      // Directives threshold | default: 50
+      n?: int,
+
+      // Callbacks that are ran whenever a Query is accepted
+      onAccept?: GraphQLArmorAcceptCallback[],
+
+      // Callbacks that are ran whenever a Query is rejected
+      onReject?: GraphQLArmorRejectCallback[],
+
+      // Do you want to propagate the rejection to the client? | default: true
```

Comment:
```suggestion
```
same for the other comments

Reply:
why ? they are descriptions about the configs !